arrow/factory.py

```diff
@@ -192,6 +197,11 @@ def get(self, *args, **kwargs):
             elif isinstance(arg, struct_time):
                 return self.type.utcfromtimestamp(calendar.timegm(arg))
 
+            # (iso calendar) -> convert then from date
+            elif isinstance(arg, tuple):
```

Comment:
Seems like the best resolution to this is to expand this case to: `elif isinstance(arg, tuple) and len(tuple) == 3:` since ISO calendar is a 3-tuple.